assets/js/base/utils/tracks.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { select } from '@wordpress/data';
+
+function isTracksAvailable() {
+	return typeof window.wcTracks.recordEvent === 'function';
+}
+
+export const recordEditorEvent = function( event, props ) {
+	if ( ! isTracksAvailable ) {
```

Comment:
Should be calling a function here?

Reply:
Yes!! Good spotting, fixed in 05fe5a98b42bd1fe61698c68435b57321180aa9d